lib/views/pages/auth_page.dart

```diff
@@ -55,6 +57,20 @@ class _AuthPageState extends State<AuthPage> {
     }
   }
 
+  Future<void> _submit2(AuthController model) async {
```

Comment:
The name is not right and doesn't describe its functionality of it.
You can review this guide:
https://dart.dev/guides/language/effective-dart/style